### PR TITLE
cpuflags: Batch of little fixes

### PR DESCRIPTION
**Pull request by ldoktor**
This PR fixes (at least) the `cpuflags..qemu_test_migration_with_additional_flags` test and adds some little cleanups.

### CODE REVIEW

File: qemu/tests/cpuflags.py [hunk 1]
```diff
@@ -404,7 +404,7 @@ def install_cpuflags_test_on_vm(vm, dst_dir):
         vm.copy_files_to(cpuflags_src, dst_dir)
         session.cmd("sync")
         session.cmd("cd %s; make EXTRA_FLAGS='';" %
-                    os.path.join(dst_dir, "cpu_flags"))
+                    os.path.join(dst_dir, "src"))
```
Comment (hereischen) [inline qemu/tests/cpuflags.py]: why not use a variable for the path?

File: qemu/tests/cpuflags.py [hunk 2]
```diff
@@ -510,9 +510,12 @@ class Test_temp(MiniSubtest):
...
         def clean(self):
             logging.info("cleanup")
-            if (hasattr(self, "vm")):
-                vm = getattr(self, "vm")
+            vm = getattr(self, "vm", None)
+            if vm:
                 vm.destroy(gracefully=False)
+            clone = getattr(self, "clone", None)
+            if clone:
+                clone.destroy(gracefully=False)
```
Comment (hereischen) [inline qemu/tests/cpuflags.py]: ACK

File: qemu/tests/cpuflags.py [hunk 3]
```diff
@@ -760,10 +760,6 @@ def test(self):
                 raise error.TestFail("Stress test ended before"
                                      " end of test.")
 
-        def clean(self):
-            logging.info("cleanup")
-            self.vm.destroy(gracefully=False)
-
```
Comment (hereischen) [inline qemu/tests/cpuflags.py]: ACK

File: qemu/tests/cpuflags.py [hunk 4]
```diff
@@ -856,15 +856,10 @@ def test(self):
...
-            self.vm.__dict__ = self.clone.__dict__
-            self.clone = temp
-
-            self.vm.resume()
-            self.clone.destroy(gracefully=False)
+            self.clone.resume()
+            self.vm.destroy(gracefully=False)
 
-            stress_session = self.vm.wait_for_login()
+            stress_session = self.clone.wait_for_login()
```
Comment (hereischen) [inline qemu/tests/cpuflags.py]: ACK

File: qemu/tests/cpuflags.py (hunk 1 above)
Comment (ldoktor) [reply qemu/tests/cpuflags.py]: this commit is a bugfix, not a style-fix. Anyway I don't really see an urgent need to turn this into a variable, usually this should not be changed. The reason it changed is that files got moved within sources.

**Pull request merged**